docs/essentials/secure-storage.md

```diff
@@ -12,6 +12,28 @@ ms.date: 05/04/2018
 
 The **SecureStorage** class helps securely store simple key/value pairs.
 
+## Getting Started
+
+To access the **SecureStorage** functionality the following platform specific setup is required.
+
+# [Android](#tab/android)
+
+No additional setup required.
+
+# [iOS](#tab/ios)
+
+When developing on the iOS Simulator it is required to enable the Keychain entitlement and add a Keychain group with the application's package identifier.
```

Comment:
- Is this necessary only for the simulator? Or for physical devices as well?
- "When developing on the iOS simulator, enable the **Keychain** entitlement and add a keychain access group for the application's bundle identifier."

Reply:
From my understanding only simulator.. :( as i have it working on phone jsut fine.. @jtaubensee @mattleibow